Frontend/ui-driver/src/Flow.purs

```diff
@@ -2782,37 +2786,64 @@ updateBannerAndPopupFlags = do
-      subscriptionConfig = appConfig.subscriptionConfig
-      freeTrialDays = fromMaybe 0 getDriverInfoResp.freeTrialDaysLeft
-      shouldShowPopup = getValueToLocalStore APP_SESSION_TRACK_COUNT == "true" && getValueToLocalNativeStore IS_RIDE_ACTIVE == "false" && (isOnFreeTrial FunctionCall || (pendingTotalManualDues /= 0.0)) && getDriverInfoResp.subscribed && appConfig.subscriptionConfig.enableSubscriptionPopups
-      autoPayStatus = getAutopayStatus getDriverInfoResp.autoPayStatus
-      autopayBannerType = if subscriptionConfig.enableSubscriptionPopups then
-                            case autoPayNotActive, isOnFreeTrial FunctionCall, (pendingTotalManualDues /= 0.0) of
-                                true, true, _  -> FREE_TRIAL_BANNER
-                                _, false, true -> do
-                                            if pendingTotalManualDues < subscriptionConfig.lowDuesLimit then LOW_DUES_BANNER
-                                            else if pendingTotalManualDues >= subscriptionConfig.lowDuesLimit && pendingTotalManualDues < subscriptionConfig.highDueWarningLimit then CLEAR_DUES_BANNER
-                                            else if pendingTotalManualDues >= subscriptionConfig.highDueWarningLimit && pendingTotalManualDues < subscriptionConfig.maxDuesLimit then DUE_LIMIT_WARNING_BANNER
-                                            else NO_SUBSCRIPTION_BANNER
-                                true, _, _ -> if isNothing getDriverInfoResp.autoPayStatus then NO_SUBSCRIPTION_BANNER else SETUP_AUTOPAY_BANNER
-                                _, _, _        -> NO_SUBSCRIPTION_BANNER
-                          else NO_SUBSCRIPTION_BANNER
-      subscriptionPopupType = case isOnFreeTrial FunctionCall, autoPayNotActive, shouldShowPopup of
-                                  true, _, true -> case freeTrialDays of
-                                                      _ | freeTrialDays == 3 || freeTrialDays == 2 || freeTrialDays == 1 -> FREE_TRIAL_POPUP
-                                                      _ -> NO_SUBSCRIPTION_POPUP
-                                  false, _, true -> if pendingTotalManualDues >= subscriptionConfig.maxDuesLimit then NO_SUBSCRIPTION_POPUP else LOW_DUES_CLEAR_POPUP
-                                  _, _, _        -> NO_SUBSCRIPTION_POPUP
-      shouldMoveDriverOffline = (withinTimeRange "12:00:00" "23:59:59" (convertUTCtoISC(getCurrentUTC "") "HH:mm:ss"))
-      moveDriverToOffline = (getValueToLocalStore MOVED_TO_OFFLINE_DUE_TO_HIGH_DUE == "") 
-                              && shouldMoveDriverOffline 
-                                && appConfig.subscriptionConfig.moveDriverToOfflineInHighDueDaily  
-                                  && getValueToLocalNativeStore IS_RIDE_ACTIVE == "false"
-                                    && pendingTotalManualDues >= subscriptionConfig.highDueWarningLimit
-                                      && getDriverInfoResp.mode /= Just "OFFLINE"
+  let
```

Comment:
is it just formatting or you did any change too in this code?

Reply:
Formatting and change also

`, waitTimeStatus = RC.waitTimeConstructor $ getValueToLocalStore WAITING_TIME_STATUS
, driverStatusSet = case getDriverInfoResp.mode of     Just mode -> getDriverStatusFromMode mode    Nothing -> getDriverStatus “”`